asmcomp/s390x/emit.mlp

```diff
@@ -611,7 +611,7 @@ let emit_instr i =
     | Lraise k ->
         begin match k with
         | Cmm.Raise_withtrace ->
-          `	brasl	%r14, {emit_symbol "caml_raise_exn"}\n`;
+          `	brasl	%r14, {emit_symbol "caml_raise_exn"}@PLT\n`;
```

Comment:
I think this is correct (using @PLT whether pic_code is requested or not), but it would be more consistent with the remainder of this file to do `emit_call "caml_raise_exn"`.